leda/Setup.md

```diff
@@ -18,69 +18,83 @@ limitations under the License.
 
 SPDX-License-Identifier: Apache-2.0
 -->
-# Setting Up a Leda Image
+The components of the Fleet Management blueprint fall into two categories: the *kuksa.val Databroker*,
+*CSV Provider* and *FMS Forwarder* components are supposed to run in the vehicle, whereas the remaining
+components are supposed to run in a (cloud) back end to which the vehicle is connected via (public internet)
+networking infrastructure.
 
-The setup for this FMS scenario assumes a split into components running in the vehicles and components
-running in a (cloud) backend. The following page describes how to configure a Leda instance to become part
-of this FMS setup. More precisely, we deploy the containers for the feedercan, the fms-forwarder,
-and the Kuksa.val databroker with an FMS-specific vehicle model.
+All of the components can be run on a single Docker host as described in the [top level README](../README.md). However, the following sections describe a more realistic deployment scenario in which
+[Eclipse Leda](https://eclipse-leda.github.io/leda/) is used as the vehicle runtime environment.
 
-## Run Leda
-The following steps assume a running instance of Eclipse Leda. For more details see the
-[Eclipse Leda Getting Started](https://eclipse-leda.github.io/leda/docs/general-usage/)
```

Comment:
I would still reference this sub-page of the Eclipse documentation, since it can be helpful for newcomers to Eclipse Leda to have a pointer to the different installation options of Eclipse Leda (QEMU, Pi4, Docker, etc.)

Reply:
It is still referenced in the *Start In-Vehicle Components* section ...

Reply:
I See, then I would have mentioned the tested Leda version down there as well. But the current version of mentioning the version here works for me too.